share/src/main/java/io/zensoft/share/service/jobkg/DefaultJobKgVacancyPublisherService.java

```diff
@@ -32,16 +34,21 @@ public DefaultJobKgVacancyPublisherService(JobKgAuthorizationService authorizati
     @Override
     public void publish(Vacancy vacancy) throws PublicationFailedException {
         try {
+            log.info("Calling authorize() method");
             String cookie = authorizationService.authorize();
+            log.info("Successfully authorized");
```

Comment:
guess `log.info("Successfully authorized");` must be in `JobKgAuthorizationService.authorize()` implementation
i think it's not a good idea to detach such kind of logs from methods